docs/src/extend/custom-processors.md

```diff
@@ -102,19 +139,41 @@ See [Specify a Processor](../use/configure/plugins#specify-a-processor) in the P
 
 ## File Extension-named Processor
 
-If a processor name starts with `.`, ESLint handles the processor as a **file extension-named processor** especially and applies the processor to the kind of files automatically. People don't need to specify the file extension-named processors in their config files.
+If a custom processor name starts with `.`, ESLint handles the processor as a **file extension-named processor**. ESLint applies the processor to files with that filename extension automatically. Users don't need to specify the file extension-named processors in their config files.
 
 For example:
 
 ```js
 module.exports = {
     processors: {
         // This processor will be applied to `*.md` files automatically.
```

Comment:
@btmills just double-checking: is this still true? I seem to recall we might have removed automatically applied processors? Or maybe I'm just imagining it?

Reply:
It's at least still true under `.eslintrc` ([implementation](https://github.com/eslint/eslintrc/blob/bd17b5a39b4b81467e6ad2850eab8f63a46afa8a/lib/config-array-factory.js#L1112-L1147), [tests](https://github.com/eslint/eslint/blob/75df535681d15d7d685468d637945a200301f9ee/tests/lib/eslint/eslint.js#L2947)). I don't know for sure that it's not part of flat config, but the tests all [explicitly specify `processor`](https://github.com/eslint/eslint/blob/75df535681d15d7d685468d637945a200301f9ee/tests/lib/eslint/flat-eslint.js#L2624), and I can't find evidence of the back compat logic being carried over.